patches/ui-shell_dialogs-execute_select_file_win.cc.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/ui/shell_dialogs/execute_select_file_win.cc b/ui/shell_dialogs/execute_select_file_win.cc
+index 5ef377bcbd69f2f2160aeed6ec4aa9fe3f20275c..ef56ddbe38b42410d7d6571796e025f3059c9c1a 100644
+--- a/ui/shell_dialogs/execute_select_file_win.cc
++++ b/ui/shell_dialogs/execute_select_file_win.cc
+@@ -362,6 +362,7 @@ bool ExecuteSelectSingleFile(HWND owner,
+                              std::vector<base::FilePath>* paths) {
+   // Note: The title is not passed down for historical reasons.
+   // TODO(pmonette): Figure out if it's a worthwhile improvement.
++  BRAVE_EXECUTE_SELECT_SINGLE_FILE
+   return RunOpenFileDialog(owner, std::u16string(), std::u16string(),
+                            default_path, filter, 0, filter_index, paths);
+ }
+@@ -376,12 +377,14 @@ bool ExecuteSelectMultipleFile(HWND owner,
+ 
+   // Note: The title is not passed down for historical reasons.
+   // TODO(pmonette): Figure out if it's a worthwhile improvement.
++  BRAVE_EXECUTE_SELECT_MULTIPLE_FILE
+   return RunOpenFileDialog(owner, std::u16string(), std::u16string(),
+                            default_path, filter, dialog_options, filter_index,
+                            paths);
+ }
+ 
+ bool ExecuteSaveFile(HWND owner,
++                     const std::u16string& title,
+                      const base::FilePath& default_path,
+                      const std::vector<FileFilterSpec>& filter,
+                      const std::wstring& def_ext,
+@@ -396,6 +399,7 @@ bool ExecuteSaveFile(HWND owner,
+ 
+   // Note: The title is not passed down for historical reasons.
+   // TODO(pmonette): Figure out if it's a worthwhile improvement.
++  BRAVE_EXECUTE_SAVE_FILE
+   return RunSaveFileDialog(owner, std::u16string(), default_path, filter,
+                            dialog_options, def_ext, filter_index, path);
+ }
+@@ -421,7 +425,7 @@ void ExecuteSelectFile(
+       break;
+     case SelectFileDialog::SELECT_SAVEAS_FILE: {
+       base::FilePath path;
+-      if (ExecuteSaveFile(owner, default_path, filter, default_extension,
++      if (ExecuteSaveFile(owner, title, default_path, filter, default_extension,
```

Comment:
given that these patches will be eliminated when new Chromium will arrive, this is lgtm.